examples/mayavi/interactive/adjust_cropping_extents.py

```diff
@@ -14,9 +14,11 @@
 are modified to propagate them to the filter. For more information
 on creating GUIs with Traits:
 
-    http://code.enthought.com/projects/traits/docs/html/tutorials/traits_ui_scientific_app.html
+    http://docs.enthought.com/traitsui
```

Comment:
This tutorial is now part of the TraitsUI documentation: http://docs.enthought.com/traitsui/tutorials/traits_ui_scientific_app.html